Pipfile

```diff
@@ -28,4 +28,4 @@ allow_prereleases = true
 start = "python opensir-cli"
 lint = "pylint **/*.py"
 test = "pytest"
-doc = "sphinx-build -M html . _build -W"
```

Comment:
Does this mean we can remove the `_build` entry from the gitignore?

Reply:
Yes, you are right. We can remove it